packages/app/src/app/pages/Sandbox/Editor/Header/index.tsx

```diff
@@ -154,7 +154,10 @@ const Header = ({ store, signals, zenMode }: Props) => {
 
       {sandbox.owned && (
         <Centered style={{ margin: '0 3rem' }}>
-          <CollectionInfo isLoggedIn={store.isLoggedIn} sandbox={sandbox} />
+          <CollectionInfo
+            isLoggedIn={store.isLoggedIn}
+            sandbox={{ ...sandbox }}
```

Comment:
Could you leave a comment as to why you do this? Also, maybe `sandbox.toJS()` works too.

Reply:
Will check that out, and shall leave a comment too!